.github/ISSUE_TEMPLATE/feel-feature-request.yml

```diff
@@ -0,0 +1,43 @@
+name: FEEL Feature request
+description: Suggest an idea or general improvement.
+labels: ["kind/feature", "area/feel", "component/feel-scala", "component/feel-js"]
+assignees: ["saig0"]
+body:
+  - type: textarea
+    id: description
+    attributes:
+      label: Problem you would like to solve
+      description: A clear and concise description of what the problem is.
+      placeholder: 'Example: I am always frustrated when [...]'
+    validations:
+      required: true
+  - type: textarea
+    id: solution-proposed
+    attributes:
+      label: Proposed solution
+      description: Describe the solution you would like to see implemented. Link to the DMN specification if possible.
+    validations:
+      required: true
+  - type: dropdown
+    id: request-type
+    attributes:
+      label: Type of feature request
+      description: Classify the type of feature request.
+      multiple: false
+      options:
+        - Existing feature of the DMN specification
+        - New custom extension (built-in function)
```

Comment:
💭 I imagine there are other feature requests, unrelated to the language like logging, configuration, etc.